boot/LoadLinux.c

```diff
@@ -357,28 +363,24 @@ CONFIGENTRY *LoadConfigCD(int cdromId) {
 			else if (BootIso9660GetFile(cdromId,"/linuxboot.cfg", (u8 *)KERNEL_SETUP, 0x800)>0) {
 				break;
 			}
+			// Allow to cancel CD boot with button 'B'
+			else if (risefall_xpad_BUTTON(TRIGGER_XPAD_KEY_B) == 1) {
+				// Close DVD tray and return to main menu
+				I2CTransmitWord(0x10, 0x0c01);
```

Comment:
Consider making this self-documenting by moving it to a function

Reply:
Refactored into a function.